From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: rbash-manpage

# DP: doc/rbash.1: fix bash(1) reference

# DP: doc/rbash.1: fix bash(1) reference
---
 doc/rbash.1 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/rbash.1 b/doc/rbash.1
index 39309b4..1a20b04 100644
--- a/doc/rbash.1
+++ b/doc/rbash.1
@@ -3,6 +3,6 @@
 rbash \- restricted bash, see \fBbash\fR(1)
 .SH RESTRICTED SHELL
 .nr zY 1
-.so bash.1
+.so man1/bash.1
 .SH SEE ALSO
 bash(1)
